cmd/wasm-client/main.go

```diff
@@ -385,7 +413,7 @@ func (w *wasmClient) HasPermissions(_ js.Value, args []js.Value) interface{} {
 
 	// Check that the macaroon contains each of the required permissions
 	// for the given URI.
-	return js.ValueOf(hasPermissions(macOps, ops))
```

Comment:
nit: A little confusing to me that this function is called `HasPermissions` and is also calling a helper function called `hasPermissions`. Is there a specific reason why these two functions are named this way?

Reply:
so the one is an exported method on the `wasmClient` and the other is an un-exported helper function.

Reply:
so the exported method is like: "does this wasm client have the permissions required for this URI"

and the unexported function is "do these list of ops have all the perms required to satisfy the required list of ops"

Reply:
That makes sense, but it's still a little uncanny to me seeing functions named the same thing with one uppercase while the other is lowercase. Is that a common pattern in Go?

Reply:
yeah I would say it is pretty common in go :) Very happy for you to change the name of the helper if you would prefer! Just dont want to include that in this PR since the diff is out of scope here